packages/compat/src/v1-app.ts

```diff
@@ -290,11 +322,71 @@ export default class V1App implements V1Package {
         })
       );
     }
-    return mergeTrees(trees, { overwrite: true });
+
+    const tree = mergeTrees(trees, { overwrite: true });
+
+    const outputGroups: Group[] = [
+      // scripts
+      {
+        outputFiles: this.app._scriptOutputFiles,
+        implicitKey: '_implicitScripts',
+        vendorOutputPath: this.app.options.outputPaths.vendor.js,
+      },
+      // styles
+      {
+        outputFiles: this.app._styleOutputFiles,
+        implicitKey: '_implicitStyles',
+        vendorOutputPath: this.app.options.outputPaths.vendor.css,
+      },
+    ];
+
+    const concatentations = [];
+
+    // support: app.import / outputFile / using
+    for (let entry of outputGroups) {
+      const { outputFiles, implicitKey, vendorOutputPath } = entry;
+      for (let importPath of Object.keys(outputFiles)) {
+        const headerFiles = outputFiles[importPath];
+
+        if (headerFiles.length === 0) {
+          // something went really wrong
+          throw new Error('Embroider: EWUT');
```

Comment:
Interesting, looks like we do hit this. Investigating...

```
 FAIL  test-packages/static-app-rooturl.test.js (5.506s)
  ● static-app-classic
    Command failed: ember build
    Embroider: EWUT
```

Reply:
Apparently the `static-app` hits this, specifically when `importPath === vendorOutputPath`